### PR TITLE
Browse animation of catalog items and nav

### PR DESCRIPTION
**Pull request by sg00dwin**
@spadgett

### CODE REVIEW

**Comment by spadgett** [comment]
@sg00dwin I have to say this is so much better. Thanks for working on this. It's a lot more polished.

I noticed some strangeness when filtering items using this input:

openshift web console 2017-06-20 20-14-44

It already triggered an animation my with previous PR (my bug), but it seems a little worse with these changes.

**Comment by sg00dwin** [comment]
@spadgett updated pr addresses filter transition issues

File: src/styles/animations.less [hunk 1]
```diff
@@ -34,3 +34,12 @@
     transform: translateY(-50px);
   }
 }
+
+@keyframes catalogItemFade {
```
Comment (rhamilto) [inline src/styles/animations.less]: Nit:  alpha (this rule should come first in the file)

File: src/styles/services-view.less [hunk 2]
```diff
@@ -223,17 +230,16 @@ services-view,
...
         }
       }
 
-      &.active.items-shown {
-        .services-sub-category-tab:after,
-        .services-items {
+      &.active {
+        .services-sub-category-tab:after {
           opacity: 1;
+          animation: catalogItemFade 200ms ease-out;
```
Comment (rhamilto) [inline src/styles/services-view.less]: Nit:  alpha

File: src/styles/services-view.less [hunk 3]
```diff
@@ -340,13 +360,59 @@ services-view,
...
+      }
+
+      &.ng-leave {
+        transition: 200ms ease-in opacity;
+        opacity: 1;
+        &.ng-leave-active {
+          opacity: 0;
+        }
+      }
+      &.ng-enter {
```
Comment (rhamilto) [inline src/styles/services-view.less]: Nit:  alpha

**Comment by rhamilto** [comment]
I'm seeing an unnecessary scrollbar now.  That is, the height of the items are is much larger than it needs to be.

screen shot 2017-06-22 at 10 57 31 am

Related, the subcategories background doesn't extend all the way to the bottom.
screen shot 2017-06-22 at 10 59 51 am

**Comment by sg00dwin** [comment]
tested across Chrome, FF, Safari, IE
@rhamilto  Found a minor glitch on Safari & IE that I'll be pushing a fix for shortly

**Comment by sg00dwin** [comment]
@rhamilto pushed changes to address comments. Except for the excessive height issue, which appears to be in master.

File: src/styles/services-view.less [hunk 4]
```diff
@@ -223,16 +230,15 @@ services-view,
...
         .services-sub-category-tab-image img {
           opacity: 1;
         }
       }
 
-      &.active.items-shown {
-        .services-sub-category-tab:after,
-        .services-items {
+      &.active {
+        .services-sub-category-tab:after {
```
Comment (rhamilto) [inline src/styles/services-view.less]: These should be part of the rule at line 220, no?  Line 231 sets the opacity to 0, but this rule sets it back to 1.

File: src/styles/services-view.less [hunk 5]
```diff
@@ -340,15 +359,57 @@ services-view,
...
+        bottom: 0;
+        left: 0;
+        position: absolute;
+        right: 0;
+        top: 80px;
+      }
+    }
+
+    .services-no-sub-categories.ng-leave.ng-leave-active {
+          // fade out inner items
```
Comment (rhamilto) [inline src/styles/services-view.less]: Nit:  indentation.  I recommend setting Atom to use spaces instead of tabs.

**Comment by sg00dwin** [comment]
@rhamilto pr updated

**Pull request merged**